src/tests/functional/plugin/cpu/single_layer_tests/deformable_convolution.cpp

```diff
@@ -262,157 +249,317 @@ std::vector<ngraph::op::PadType> padTypes = {
         ::testing::Values(std::vector<ptrdiff_t>({0, 0})),  // pad. end
         ::testing::Values(std::vector<size_t> {1, 1}),  // strides
         ::testing::Values(std::vector<size_t> {1, 1})  // dilations
 );
-const auto spParams4 = ::testing::Combine(
-        ::testing::Values(1),  // batch
-        ::testing::Values(std::vector<size_t>({3, 2})),  // in. spat. shape
-        ::testing::Values(std::vector<size_t>({2, 1})),  // off. spat. shape
-        ::testing::Values(std::vector<size_t>({2, 2})),  // ker. spat. shape
-        ::testing::ValuesIn(padTypes),  // pad. type
-        ::testing::Values(std::vector<ptrdiff_t>({0, 0})),  // pad. begin
-        ::testing::Values(std::vector<ptrdiff_t>({0, 0})),  // pad. end
-        ::testing::Values(std::vector<size_t> {1, 1}),  // strides
-        ::testing::Values(std::vector<size_t> {1, 1})  // dilations
-);
-
-const auto chParamsSingleGr = ::testing::Combine(
-        ::testing::ValuesIn(std::vector<size_t> {1}),  // gr. 1
-        ::testing::ValuesIn(std::vector<size_t> {1, 2}),  // def. gr. 1,2
-        ::testing::ValuesIn(std::vector<size_t> {16, 32}),  // in. ch. per gr.
-        ::testing::ValuesIn(std::vector<size_t> {16, 32}));  // out. ch. per gr.
 
-const auto chParamsMulGr = ::testing::Combine(
-        ::testing::ValuesIn(std::vector<size_t> {2, 4}),  // gr. 2,4
-        ::testing::ValuesIn(std::vector<size_t> {1, 2}),  // def. gr. 1,2
-        ::testing::ValuesIn(std::vector<size_t> {3, 7}),  // in. ch. per gr.
-        ::testing::ValuesIn(std::vector<size_t> {3, 7}));  // out. ch. per gr.
+const std::vector<std::vector<size_t>> spatParams1 = {
+    {1}, // batch
+    {34, 34}, // in. spat. shape
+    {32, 32}, // off. spat. shape
+    {3, 3} // ker. spat. shape
+};
+const std::vector<std::vector<size_t>> spatParams2 = {
+    {1}, // batch
+    {3, 3}, // in. spat. shape
+    {2, 2}, // off. spat. shape
+    {2, 2} // ker. spat. shape
+};
+const std::vector<std::vector<size_t>> spatParams3 = {
+    {1}, // batch
+    {5, 5}, // in. spat. shape
+    {4, 4}, // off. spat. shape
+    {2, 2} // ker. spat. shape
+};
+const std::vector<std::vector<size_t>> spatParams4 = {
+    {1}, // batch
+    {3, 2}, // in. spat. shape
+    {2, 1}, // off. spat. shape
+    {2, 2} // ker. spat. shape
+};
+const std::vector<std::vector<size_t>> channelParamsSingleGr = {
+    {1}, // gr. 2,4
+    {1, 2}, // def. gr. 1,2
+    {16, 32}, // in. ch. per gr.
+    {16, 32} // out. ch. per gr.
+};
+const std::vector<std::vector<size_t>> channelParamsMulGr = {
+    {2, 4}, // gr. 2,4
+    {1, 2}, // def. gr. 1,2
+    {3, 7}, // in. ch. per gr.
+    {3, 7} // out. ch. per gr.
+};
 
+const std::vector<std::vector<InputShape>> dynShapeChainRef = {
+        {
+            // gr == 2, dg == 1, in_ch_per_gr == 3, out_ch_per_gr == 3
+            // {{dynamic shape}, {{static shape case1}, {static shape case2}, ...}
+            {{-1, 6, -1, -1}, {{1, 6, 3, 2}, {1, 6, 4, 3}, {1, 6, 5, 4}}},  // input 0
```

Comment:
could you also add test case when all dims include channel == -1?

and please measure tests execution time and separate test in `smoke` and `nightly`

Reply:
Already separated. Autopad case added in smoke